src/gui/tray/SyncStatus.qml

```diff
@@ -0,0 +1,97 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Style 1.0
+
+import com.nextcloud.desktopclient 1.0 as NC
+
+RowLayout {
+    id: layout
+
+    property alias model: syncStatusModel
```

Comment:
can we call it `syncStatus` ?
when reading the code I was assuming some kind of model I could display with a `ListView` instance